### PR TITLE
Remove unnecessary default values.

### PR DESCRIPTION
**Pull request by rictic**
Free startup performance improvement for carbon-location.

Also, ignore path-change events from carbon-route-converter until the location
is ready.

### CODE REVIEW

**Comment by rictic** [comment]
These were some unrelated changes that came up while talking with @sorvell today about #93

File: carbon-location.html [hunk 1]
```diff
@@ -148,11 +148,18 @@
         }
       },
 
+      ready: function() {
+        this._isReady = true;
```
Comment (cdata) [inline carbon-location.html]: Isn't there already a `_readied` value that is set by Polymer?

Comment (rictic) [inline carbon-location.html]: There is, and that's what Steve used when we were testing this, but I wasn't sure if it was kosher to  use that, as it's private to Polymer.

I did a quick search and I couldn't find any uses of it anywhere, in our elements or in other folks' code that I could easily search.

Comment (rictic) [inline carbon-location.html]: 😓 I misspelled it when I searched.  We've shipped a use of it in paper-dialog-behavior.

**Comment by cdata** [comment]
LGTM 👍

**Pull request merged**